.github/workflows/openms_ci_matrix_full.yml

```diff
@@ -547,14 +552,23 @@ jobs:
         uses: ncipollo/release-action@v1
         with: 
           bodyFile: RELEASE_TEXT_GH.md
-          tag: 2.9.1-test #${{ steps.extract_branch.outputs.RUN_NAME }}
-          draft: true
+          tag: ${{ env.RUN_NAME }}
+          draft: true #FIXME revert
```

Comment:
Draft might actually be fine. Could be a good way to double-check. But on the other hand, if something is wrong, the damage was done already, so it probably does not matter.